homeassistant/components/bsblan/climate.py

```diff
@@ -67,10 +65,12 @@ async def async_setup_entry(
 
 
 class BSBLANClimate(
-    BSBLANEntity, CoordinatorEntity[DataUpdateCoordinator[State]], ClimateEntity
+    BSBLANEntity, CoordinatorEntity["BSBLanUpdateCoordinator"], ClimateEntity
```

Comment:
So what I'm wondering, why does it inherit both the BSBLANentity and the coordinator entity, while the BSBLANentity already inherits the coordinator one

Reply:
lol indeed this makes no sense. I need to update the entity too to inherit the coordinator. Cutting the pull request in pieces is quite difficult

Reply:
I am fine with making sure that the entity inherits the coordinator in this PR